packages/asset-swapper/src/types.ts

```diff
@@ -193,9 +193,11 @@ export interface SwapQuoteOrdersBreakdown {
  */
 export interface SwapQuoteRequestOpts extends CalculateSwapQuoteOpts {
     gasPrice?: BigNumber;
-    takerAddress?: string;
     apiKey?: string;
-    intentOnFilling?: boolean;
+    rfqt?: {
+        takerAddress?: string;
```

Comment:
`takerAddress` should not be optional in this context.  When it was its own key in `SwapQuoteRequestOpts` it made sense, but now that it's within the optional `rfqt` key, it should be required.

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/264407b7072b3dec01c27c872c2f43c5af40d690